docs/examples/test_examples.py

```diff
@@ -0,0 +1,268 @@
+""" Tests for the files in nameko/docs/examples
+"""
+import os
+
+import eventlet
+from mock import call, patch
+import pytest
+import requests
+
+from nameko.containers import ServiceContainer
+from nameko.standalone.events import event_dispatcher
+from nameko.standalone.rpc import ServiceRpcProxy
+from nameko.testing.services import entrypoint_waiter
+from nameko.testing.websocket import make_virtual_socket
+
+
+@pytest.yield_fixture
+def container_factory(rabbit_config):
```

Comment:
Now might be the time to promote these fixtures out of ``conftest.py`` so we can import rather than reimplement them here.